app.py

```
@@ -131,6 +134,10 @@
 ai_service = None
 
 
+# 认证服务：凭据存放在仓库安装目录的 data/auth.json，与 HUGO_ROOT 解耦
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)